File: practices/time.py
# AT what time is it practice

time=int(input("what time is it in military time?:\n"))

if time < 11:
    print(f" its {time} good morning")
elif time > 11:
    print(f" its {time} goodafternooon")
elif time > 4:
    print(f" its {time} good evening")

File: c_programming/practice/time.c
// AT 6th time of day practice
#include <stdio.h>
#include <string.h>
int main(void){
    int time;
    printf("what hour is it in military time?: ");
    scanf("%d", &time);
    if (time >= 18)
        printf("good evening");
    else if (time >= 12)
        printf("good afternoon");
    else if (time <= 11)
        printf("good morning");
    return 0;
    

}